### PR TITLE
✨ Add Storage Modules #7

### LINKED ISSUE #7: Add Storage samples
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

It would be great, for testing purposes and for testing applications, samples and templates to generate filestorage, blobstorage, tablestorage, etc

### Describe the solution you'd like

Create new `storageaccounts` modules and, create samples in the samples folder so that anyone can trigger the sample whenever they might need.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Rabosa616**
# ✨ Add Storage Modules

- [x] You've read the [Contributor Guide](./../CONTRIBUTING.md) and [Code of Conduct](./../CODE-OF-CONDUCT.md).
- [x] You've included unit or integration tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

Summary of the changes (Less than 80 chars)

## Description

Add Storage modules:

- Storage Account
- Storage Container
- Storage Queue
- Storage Table

Fixes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,27 +71,28 @@ Installation instructions assets-iac-terraform-azure by running:
    > Some modules expects that you have resources created. e.g. a Resource Group. Then make sure you include a "depends_on"
 
 4. Execute the modules by running the following [terraform commands](https://www.terraform.io/cli/commands):
-    * login
-        * az login
+    - login
```
Comment (albertinisg) [inline README.md:74]: I would rather add these scripts using fenced code blocks and removing indentation. For example:

1. Authenticate using the Azure CLI

```
$ az login
```

2. Choose the subscription to use by doing:

```
az account set --subscription <id>
```
3. Browse to the module you want to deploy by browsing to modules/XXXX.
4. Modify the variables to match your needs at `modules/XXXX/terraform.tfvars` and/or `variables.tf`
5. initizlize your terraform configuration:

```
$ terraform init
```
6. Validate your configuration
```
terraform validate
```
7. Plan your Terraform Run
```
$ terraform plan
```
8. Apply your Terraform Configuration
```
$ terraform apply
```

**Review by albertinisg — COMMENTED** [review]
I think that, at this time, we can modify the `Modules` section in the README,md and add the new ones (with links) but also to create categories like: storage, etc

What do you think?

File: README.md (hunk 1 above)
Comment (Rabosa616) [reply README.md:74]: I would prefer to create a new Issue to refactor the readme file

**Review by albertinisg — APPROVED** [review]
Approved as per discussed :)

**Pull request merged**